src/components/todoList/TodoList.jsx

```diff
@@ -1,13 +1,29 @@
-import React from 'react';
+import React from 'react'
+import { Container, Row } from "react-bootstrap";
+import { HeaderDashboard } from "../dashboard/HeaderDashboard";
+import { useTranslation } from "react-i18next";
+import { useRecoilValue } from 'recoil';
+import { darkModeState } from '../../store/DarkModeState'
+import "../../translations/i18n";
+import "./TodoList.scss"
+import { TodoForm } from './TodoListForm'
 
-import { HeaderDashboard } from '../dashboard/HeaderDashboard';
-import '../../translations/i18n';
-
-export function TodoList() {
+export const TodoList = () => {
+  const darkModeStatus = useRecoilValue(darkModeState);
+  const { t } = useTranslation();
   return (
     <div>
-      <HeaderDashboard />
-      <h1>Add a Goal</h1>
+      <Container fluid>
+        <Row>
+          <HeaderDashboard />
+        </Row>
+        <Row>
+          <h2 className={darkModeStatus ? "myGoals-font-dark" : "myGoals-font-light"}>{t("myGoalsMessage")}</h2>
```

Comment:
Don't mix camelCase and kebab-case in class names. Class names are supposed to be written with `kebab-case`